pandas/core/generic.py

```diff
@@ -457,12 +457,39 @@ def axes(self):
 
     @property
     def ndim(self):
-        """Number of axes / array dimensions"""
+        """
+        Return an int representing the number of axes / array dimensions.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df.ndim
+        2
+
+        >>> df = pd.DataFrame({'col1': [1, 2, 3], 'col2': [4, 5, 6],
+        ...                    'col3': [7, 8, 9]})
+        >>> df.ndim
+        2
+        """
```

Comment:
Can you also add an example for Series? (and I think one for DataFrame is good enough)

Reply:
Okay, I will add.